package.json

```diff
@@ -5,14 +5,16 @@
   "main": "index.js",
   "scripts": {
     "build": "babel server -d dist --source-maps",
-    "test": "nyc --reporter=text --reporter=html mocha --timeout=3000 --recursive --exit"
+    "test": "nyc --reporter=text --reporter=html mocha --timeout=3000 --recursive --exit",
+    "devstart": "DEBUG=vale-ah* nodemon server/index.js --exec babel-node"
   },
   "author": "Andela Simulations Programme",
   "license": "MIT",
   "dependencies": {
     "@babel/polyfill": "^7.2.5",
     "body-parser": "^1.18.3",
     "cors": "^2.8.4",
+    "debug": "^4.1.1",
     "dotenv": "^6.0.0",
     "ejs": "^2.6.1",
     "errorhandler": "^1.5.0",
```

Comment:
Some packages seem to have been removed from the server entry file (ex: errorhandler). I don't think there is any reason to have it hanging around.

Reply:
#7 Removes the unnecessary packages from the package.json file